Tools/SwiftBrowser/Source/ContentView.swift

```diff
@@ -55,7 +55,97 @@ fileprivate struct ToolbarBackForwardMenuView: View {
     }
 }
 
-fileprivate struct DialogActionsView: View {
+private struct MediaCaptureStateButtonView: View {
+    struct LabelConfiguration {
+        let activeSystemImage: String
+        let mutedSystemImage: String
+    }
+
+    @Binding var captureState: WKMediaCaptureState
+
+    let configuration: LabelConfiguration
+
+    var body: some View {
+        switch captureState {
+        case .none:
+            EmptyView()
+
+        case .active:
+            Button {
+                captureState = .muted
+            } label: {
+                Label("Mute", systemImage: configuration.activeSystemImage)
+                    .labelStyle(.iconOnly)
+            }
+
+        case .muted:
+            Button {
+                captureState = .active
+            } label: {
+                Label("Unmute", systemImage: configuration.mutedSystemImage)
+                    .labelStyle(.iconOnly)
+            }
+
+        @unknown default:
+            fatalError()
+        }
+    }
+}
+
+private struct PrincipalToolbarGroup: View {
```

Comment:
You could conform this to `ToolbarContent, DynamicProperty` instead, and then use it directly in the `toolbar` below (but this is also fine).

Reply:
how would I do this? When I try simply just conforming it and nothing else, I get these errors. When I try to make `body` be `some ToolbarContent`, I get even more errors.


<img width="640" alt="image" src="https://github.com/user-attachments/assets/420b6656-0fed-4e98-bcac-f120d5ff954d" />